scholly/base/package.json

```diff
@@ -17,7 +17,7 @@
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "cross-env REACT_APP_DEBUG=true react-scripts start",
```

Comment:
Great idea for that `cross-env` package, I didn't know about that! That should help w/ env var compatibility across platforms.
I would strongly suggest placing the command `"cross-env REACT_APP_DEBUG=true react-scripts start"` under a `dev` script so that the deployment in prod still uses the real data - cuz otherwise the fake data will be used there as well.

E.g.
```
 "scripts": {
    "dev": "cross-env REACT_APP_DEBUG=true react-scripts start",
    "start": "<leave old start script>"
...
}
```

So now whoever wants to run the app in debug mode can just execute `npm run dev`.

Reply:
Ahh right. Just wondering though, in the scholly service we invoke `npm build` in the base app, does that then run `npm start`?? If it doesn't then it shouldn't matter much if we add the cross-env stuff in dev or start script, I'm guessing